app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTile.kt

```diff
@@ -75,11 +76,16 @@ class CustomPinnedTile(
         put(KEY_ID, id.toString())
     }
 
-    override fun toChannelTile(screenshotStoreWrapper: ScreenshotStoreWrapper): ChannelTile {
+    override fun toChannelTile(imageUtilityWrapper: PinnedTileImageUtilWrapper): ChannelTile {
+        val backup = imageUtilityWrapper.generatePinnedTilePlaceholder(url)
+
         return ChannelTile(
                 url = url,
                 title = title,
-                setImage = { view -> PicassoWrapper.client.load(screenshotStoreWrapper.getFileForUUID(id)).into(view) } // todo: fix scope, double check this is okay.
+                setImage = { view -> PicassoWrapper.client
+                        .load(imageUtilityWrapper.getFileForUUID(id))
```

Comment:
Does this throw anything that doesn't get caught if the UUID doesn't exist as a file? (Maybe this is already handlede before)

Reply:
Just tested, it cleanly falls back to the default.  Good callout.